config/project_params.yaml

```diff
@@ -2,7 +2,6 @@ topics:
   joint_encoder: /mcu/state/jointURDF
   imu: /mcu/state/imu
   state_estimate: /state_estimate
-  joint_states: /joint_states
```

Comment:
Good call - was anything using this?

Reply:
Nope, I added it provisionally early on, didn't realize we had a duplicate added at some point